Paper/HighDimensionalSampling.tex

```diff
@@ -18,7 +18,7 @@
 }
 
 \title{A comparison of sampling algorithms for particle astrophysics applications}
-\author{The DarkMachines High Dimensional Sampling group: ...}
+\author{The DarkMachines High Dimensional Sampling group: ... needs to include Ben Farmer and Gregory Martinez for pyscannerbit}
```

Comment:
Has the paper been moved to overleaf? Do we need to commit a note like this to the main repository?